src/components/navigation.tsx

```diff
@@ -17,11 +17,16 @@ import {locales} from '@/middleware';
 interface Props {
   locale: string;
   navigationLabels: {
+    logo: string;
     home: string;
     register: string;
     about: string;
     faq: string;
     contact: string;
+    aria: {
+      languageSelector: string;
+      openMenu: string;
+    };
```

Comment:
I don't like this code. But I'm hoping at some point, `next-intl` will have a better solution for client components. This is based on the docs of `next-intl`. I tried `{[key: string]: string}`. But this will give a typescript if you ask for specific keys. I would like to keep this like this for now. And first, see if `next-intl` will make using translations in client components easier.

Reply:
Perfect!